bin/.travis/check_code_style.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+if [[ -z "${TRAVIS_PULL_REQUEST}" || -z "${TRAVIS_PULL_REQUEST_BRANCH}" ]]
+then
+    echo "> Code Style check: missing required Travis environment variables TRAVIS_PULL_REQUEST or TRAVIS_PULL_REQUEST_BRANCH" >&2;
+    exit 1;
+fi
+
+if [[ "${TRAVIS_PULL_REQUEST}" != "false" ]]
+then
+    # limit checking to the files which were affected by a current PR
+    FILES_LIST=$(git diff --diff-filter=ACMR --name-only "${TRAVIS_PULL_REQUEST_BRANCH}" "*.php"|paste -sd ' ');
```

Comment:
- `ACMR` - added, modified, copied, renamed
- `paste -sd  '  '` - join multiple lines (one file name per line) and separate by space

In general gets the diff with respect to the base branch of the PR.